docs/self-managed/optimize-deployment/configuration/logging.md

```diff
@@ -6,11 +6,11 @@ description: "Camunda Optimize provides logging facilities that are preconfigure
 
 Camunda Optimize provides logging facilities that are preconfigured to use
 _INFO_ logging level which provides minimal output of information in log files.
-This level can be adjusted using the `environment-logback.xml` configuration file.
+This level can be adjusted using the `environment-log4j2.xml` configuration file.
 
 ## Google Stackdriver (JSON) logging
 
-To enable Google Stackdriver compatible JSON logging, set the environment variable `JSON_LOGGING=true` before starting Optimize.
+To enable Google Stackdriver compatible JSON logging, set the environment variable `OPTIMIZE_LOG_APPENDER=Stackdriver` before starting Optimize.
```

Comment:
I think this is a breaking change and should be communicated in our update guide: https://docs.camunda.io/docs/next/self-managed/operational-guides/update-guide/870-to-880/

Basically, I would see this as the user needing to do something, so we should therefore tell them what to do